src/Castle.Windsor/Castle.Windsor-VS2017.csproj

```diff
@@ -35,4 +35,10 @@
   <PropertyGroup Condition="'$(TargetFramework)'=='net45'">
     <DefineConstants>$(DefineConstants);FEATURE_PERFCOUNTERS;FEATURE_GAC</DefineConstants>
   </PropertyGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard1.5'">
```

Comment:
@eerhardt - Thanks for the explanation. 

> Where does this target framework come from? Above, this project only declares:
> <TargetFramework>net45</TargetFramework>

The intention is to multi-target similar to https://github.com/castleproject/Core/blob/refactor-build/src/Castle.Core/Castle.Core.csproj#L6 eventually. 

I am sure you can appreciate things are moving around a fair bit here, time for us is also at a premium and "chasing moved cheese" becomes rather dull after a while(sorry if this offends). My challenge was merely to understand why a dependency that potentially could be superseded by roughly 7-8 lines of code was nowhere to be found on the cli repo when I knew it was there before. This got me round to thinking is this really another "goose chase" we want at this stage. 

Your explanation has definitely helped. I will bow out of this PR and leave it to @jonorossi and @alinapopa to decide whether to take it forward.

Again thanks for taking the time to explain although I still believe we don't need it for most use cases.